lib/db/mysql/patches/patch-014-015.sql

```diff
@@ -0,0 +1,50 @@
+-- Add purge expired tokens stored procedure and index to expiresAt column
+ALTER TABLE tokens ADD INDEX idx_expiresAt (expiresAt);
```

Comment:
@vbudhram 5.6.21 in prod : https://github.com/mozilla/fxa-auth-db-mysql#prerequisites

You can switch travis to 5.6 : https://docs.travis-ci.com/user/database-setup/#MySQL-5.6